plugins/woocommerce/client/legacy/css/admin/product-editor.scss

```
@@ -0,0 +1,748 @@
+	.postbox {
+		margin-bottom: 28px;
+	}
+
+	// Description editor.
+	#postdivrich {
+		margin-top: 28px;
+		// Make it look like postbox, w/o class in no-js env.
+		background: $studio-white;
+
+		.wp-editor-wrap {
+			margin: $postbox-padding $postbox-padding 0;
+		}
+		#post-status-info {
+			margin: 0 $postbox-padding $postbox-padding;
+			width: calc(100% - 2 * $postbox-padding);
+		}
+	}
+
+	// Post/meta boxes.
+	.postbox-header {
+		padding: $postbox-padding-small $postbox-padding-small $postbox-padding-small $postbox-padding;
+		// Offset for the invisible postbox border.
+		margin: -1px -1px 0;
+		border-color: $separator-color;
+		min-height: 30px;
+
+		.hndle {
+			gap: $gap-small;
+			flex-wrap: wrap;
+		}
+	}
+	.closed .postbox-header {
+		border: 0;
+	}
+
+	.inside,
+	#commentsdiv.postbox .inside,
+	#postimagediv .inside,
+	#woocommerce-product-images .inside {
+		padding: 0 $postbox-padding;
+	}
+
+	// Separate rule, not to overspecify the above ones with id selector.
+	#poststuff {
+		h2.postbox-header {
+			padding: $postbox-padding-small $postbox-padding;
+		}
+		.hndle {
+			padding: 0;
+		}
+
+		.inside {
+			margin: $postbox-padding 0;
+		}
+		.postbox .inside {
+			// Offset for the invisible postbox border.
+			margin: $postbox-padding -1px calc($postbox-padding - 1px) -1px;
+		}
+	}
+
+	.wp-editor-tools {
+		display: flex;
+		flex-wrap: wrap;
+		justify-content: space-between;
+	}
+
+	#wp-content-editor-tools {
+		// Reduce FOUC before `woocommerce-product-description` class is attached.
+		background: none;
+		padding-top: 0;
+	}
+
+	// Rich editor tabs.
+	.wp-editor-tabs {
+		display: flex;
+		align-self: flex-end;
+	}
+
+	.wp-switch-editor {
+		float: none;
+		background: none;
+		margin: 0;
+		padding: 10px $gap;
+		color: var(--Alias-text-text-tertiary, $alias-text-tertiary);
+		border-width: 0 0 2px 0;
+		border-bottom-color: transparent;
+	}
+	.html-active .switch-html,
+	.tmce-active .switch-tmce {
+		border-bottom-color: var(--wp-admin-theme-color, $studio-wordpress-blue);
+		color: var(--Alias-text-text, $alias-text);
+	}
+
+	.wp-media-buttons {
+		display: flex;
+		align-self: flex-start;
+		flex-wrap: wrap;
+		gap: $gap-small;
+		padding-bottom: $gap;
+
+		.button {
+			margin: 0;
+			padding: 0 $gap-smaller;
+		}
+	}
+	.wp-editor-tools::after {
+		content: none;
+		display: none;
+	}
+
+	#postexcerpt > .postbox-header > .hndle > .woocommerce-help-tip,
+	#wp-content-media-buttons > .woocommerce-help-tip {
+		margin: 0;
+	}
+
+	// Product/coupon data options panel.
+	.woocommerce_options_panel {
+		p,
+		fieldset.form-field,
+		.downloadable_files {
+			// Use margin as gap and horizontal padding.
+			// Accomodate for p trick with label that uses padding as margin
+			// that adds 12 to the left and 20 to the right
+			margin: $gap calc(48px - 20px) $gap calc(48px - 12px);
+
+			@media only screen and (max-width: $break-wide) {
+				margin: $gap calc(gap-larger - 20px) $gap calc($gap-larger - 12px);
```

Variable is missing the starting `$`.

```suggestion
				margin: $gap calc($gap-larger - 20px) $gap calc($gap-larger - 12px);
```